sirepo/template/radia.py

```diff
@@ -1140,6 +1145,31 @@ def _update_geom_from_undulator(geom, und, beam_axis):
     magnet_transverse_ctr = magnet_dim_half.width / 2 - \
                             (gap_offset + magnet_dim_half.height + gap_half_height)
 
+    obj_props = PKDict(
```

Comment:
This seems repetitive and could be collapsed especially if the code above used more dicts and/or lists.